From 482235b17b3a41433d6d26c9c0b59c7a9789f374 Mon Sep 17 00:00:00 2001
From: Damien Zammit <damien@zamaudio.com>
Date: Tue, 26 Jan 2016 14:06:26 +1100
Subject: [PATCH] mb/intel/d510mo: Explicitly select NIC on PCI in devicetree

While the board configuration still works without this,
It's nicer to have the device statically defined since
the NIC is hardwired to the board.

Change-Id: Ic6682865dd17672c3782bfba9511cd120d1657c1
Signed-off-by: Damien Zammit <damien@zamaudio.com>
---
 src/mainboard/intel/d510mo/devicetree.cb | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/mainboard/intel/d510mo/devicetree.cb b/src/mainboard/intel/d510mo/devicetree.cb
index c0f38de..c5b885f 100644
--- a/src/mainboard/intel/d510mo/devicetree.cb
+++ b/src/mainboard/intel/d510mo/devicetree.cb
@@ -43,7 +43,9 @@ chip northbridge/intel/pineview		# Northbridge
       register "gpe0_en" = "0x20000040"
 
       device pci 1b.0 on end		# Audio
-      device pci 1c.0 on end		# PCIe 1
+      device pci 1c.0 on		# PCIe 1
+        device pci 0.0 on end		# NIC
+      end
       device pci 1c.1 on end		# PCIe 2
       device pci 1c.2 on end		# PCIe 3
       device pci 1c.3 on end		# PCIe 4
-- 
1.9.1

